### DISCUSSION TITLE
Connection Refused updating Lists

### DISCUSSION THREAD

**Discussion by smegoff** [blocklistproject/Lists#304] [Q&A] [answered]
Hi There,

Just trying to understand an outstanding issue I had using your block lists.

 [i] Target: https://blocklistproject.github.io/Lists/alt-version/abuse-nl.txt
  [✗] Status: Connection Refused
  [✗] List download failed: no cached list available

The same applies for original lists too.

I did a pihole-d and it didnt allude to any issues as such.

I can browse to the list fine. Pi-Hole wont update it.

TIA

Sean

**Comment by Trainax** [comment]
I tried downloading again the lists and I could not reproduce this issue.

Is the time on your system correct? Sometimes connections can be refused if the time is not set correctly.

Also can you check if you are running the latest Pi-hole version? You can do it by launching the `pihole -up` command in the terminal

**Comment by smegoff** [comment]
> 
> 
> I tried downloading again the lists and I could not reproduce this issue.
> 
> Is the time on your system correct? Sometimes connections can be refused if the time is not set correctly.
> 
> Also can you check if you are running the latest Pi-hole version? You can do it by launching the `pihole -up` command in the terminal

Thanks for the reply. 

Time is correct. 
Pi-Hole is all up to date. I run two and use pi-hole rsync to keep the lists adlists in sync. 

I have two other lists that seem to work OK.

**Comment by smegoff** [comment]
> 
> 
> I tried downloading again the lists and I could not reproduce this issue.
> 
> Is the time on your system correct? Sometimes connections can be refused if the time is not set correctly.
> 
> Also can you check if you are running the latest Pi-hole version? You can do it by launching the `pihole -up` command in the terminal

for reference.

[i] Neutrino emissions detected...
  [✓] Pulling blocklist source list into range

  [✓] Preparing new gravity database
  [i] Using libz compression

  [i] Target: https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts
  [✓] Status: Retrieval successful
  [i] Analyzed 83269 domains
  [i] List stayed unchanged

  [i] Target: https://raw.githubusercontent.com/mhhakim/pihole-blocklist/master/list.txt
  [✓] Status: Retrieval successful
  [i] Analyzed 792004 domains
  [i] List stayed unchanged

  [i] Target: https://blocklistproject.github.io/Lists/alt-version/abuse-nl.txt
  [✗] Status: Connection Refused
  [✗] List download failed: no cached list available

**Comment by Trainax** [comment, marked as answer]
> 
> 
> > I tried downloading again the lists and I could not reproduce this issue.
> > Is the time on your system correct? Sometimes connections can be refused if the time is not set correctly.
> > Also can you check if you are running the latest Pi-hole version? You can do it by launching the `pihole -up` command in the terminal
> 
> for reference.
> 
> [i] Neutrino emissions detected...
> [✓] Pulling blocklist source list into range
> 
> [✓] Preparing new gravity database
> [i] Using libz compression
> 
> [i] Target: https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts
> [✓] Status: Retrieval successful
> [i] Analyzed 83269 domains
> [i] List stayed unchanged
> 
> [i] Target: https://raw.githubusercontent.com/mhhakim/pihole-blocklist/master/list.txt
> [✓] Status: Retrieval successful
> [i] Analyzed 792004 domains
> [i] List stayed unchanged
> 
> [i] Target: https://blocklistproject.github.io/Lists/alt-version/abuse-nl.txt
> [✗] Status: Connection Refused
> [✗] List download failed: no cached list available

Try using the raw.githubusercontent link instead of the GitHub Pages one to do a test to see if the problem is in fetching from Pages

https://raw.githubusercontent.com/blocklistproject/Lists/master/alt-version/abuse-nl.txt

**Comment by smegoff** [comment]
If I use 
https://raw.githubusercontent.com/blocklistproject/Lists/master/abuse.txt

It works fine :-)

**Comment by Trainax** [comment]
> 
> 
> If I use
> https://raw.githubusercontent.com/blocklistproject/Lists/master/abuse.txt
> 
> It works fine :-)

This doesn't look like a problem which can be fixed by changing something in the txt files in this repo as it works if you use the raw.githubusercontent link...

I would suggest you to continue using the raw.githubusercontent link in your Pi-hole until the cause of the problem is found
